events/loginEvent.py

```diff
@@ -90,6 +93,21 @@ async def handle(web_handler: AsyncRequestHandler) -> tuple[str, bytes]:  # toke
             # Invalid password
             raise exceptions.loginFailedException()
 
+        try:
+            # we have a user ID we can rely on, allow further processing of login body
+            bancho_login_request = {
+                "login_body": web_handler.request.body.decode(),
+                "user_id": userID,
+            }
+
+            for routing_key in settings.BANCHO_LOGIN_ROUTING_KEYS:
+                await glob.amqp_channel.default_exchange.publish(
+                    aio_pika.Message(body=orjson.dumps(bancho_login_request)),
+                    routing_key=routing_key,
+                )
+        except Exception:  # don't allow publish failure to block login
+            logger.exception("Failed to send bancho login request through AMQP")
```

Comment:
maybe worth making it less urgent from the logs pov
```suggestion
            logger.warning("[Non-blocking] Failed to send bancho login request through AMQP", exc_info=True)
```